lib/wash/exe/chrome.js

```diff
@@ -13,16 +13,112 @@
 // limitations under the License.
 
 import AxiomError from 'axiom/core/error';
+import Completer from 'axiom/core/completer';
 import chromeAgentClient from 'wash/chrome_agent_client';
 
 /** @typedef JsExecuteContext$$module$axiom$fs$js$execute_context */
 var JsExecuteContext;
 
+/** @const @private */
+var STDIN_ = '$STDIN';
+/** @const @private */
+var STDIN_RE_ = new RegExp(STDIN_);
```

Comment:
Does this work?  I would think that you'd need to escape the `$` as `\\$` to match `"$STDIN"`, otherwise the $ will act as end-of-line.